gpu/examples/compute/compute.go

```diff
@@ -30,7 +32,19 @@ type Data struct {
 }
 
 func main() {
-	gpu.Debug = true
+	// errors.Log1(jsfs.Config(js.Global().Get("fs"))) // needed for printing etc to work
+	// time.Sleep(1 * time.Second)
+	// b := core.NewBody()
+	// bt := core.NewButton(b).SetText("Run Compute")
+	// bt.OnClick(func(e events.Event) {
+	compute()
+	// })
+	// b.RunMainWindow()
+	// select {}
```

Comment:
Clean this up?

Reply:
not now -- need in future for web testing.